frontends/election-manager/test/helpers/api.ts

```diff
@@ -0,0 +1,28 @@
+import type { Api } from '@votingworks/admin'; // eslint-disable-line vx/gts-no-import-export-type
+import { createMockClient, MockClient } from '@votingworks/grout-test-utils';
+import { DippedSmartCardAuth } from '@votingworks/types';
+
+export type MockApiClient = Omit<MockClient<Api>, 'getAuthStatus'> & {
+  // Because this is polled so frequently, we opt for a standard jest mock instead of a
+  // libs/test-utils mock since the latter requires every call to be explicitly mocked
```

Comment:
One limitation of the new mocking approach - not a big deal and easy enough to work around!

Reply:
Makes sense. I've been noodling on this and I have another idea for how to work around it: In test, configure the polling interval to be something really large, then use fake timers to advance whenever we are testing something that depends on an auth status change.

But happy to try this approach for now and see how it goes. We could potentially build some sort of support for this type of mocking into the mock client if we like it.

Reply:
Also, I'd be curious to hear how things went with the new mocking approach. Did you run into any issues? Did it make anything easier?

Reply:
Oh nice, very reasonable alternative! I like the current approach since it requires little mental overhead and feels like a very direct way of getting to what we need for tests. But am open to experimentation down the road

Reply:
And in terms of the new mocking approach, I found the error messages when calls didn't match up to be very helpful as I was debugging failing tests!